backend/src/main/java/com/springBoot/fruits_ecommerce/models/Role.java

```diff
@@ -0,0 +1,32 @@
+package com.springBoot.fruits_ecommerce.models;
+
+import com.springBoot.fruits_ecommerce.enums.RoleName;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
```

Comment:
@AllArgsConstructor is not recommended here, lets use Static Constructor instead